robocopkaka/bootcamp-29-project
# #157930865 Add tests for actions and reducers

#### What does this PR do?
add tests for actions and reducers
#### Description of Task to be completed?
* add tests for actions
* add tests for reducers
* modify actions to pass tests
* modify reducers to pass tests
#### How should this be manually tested?
* clone this repo
* cd into the project folder
* run `npm install`
* run `npm run test:jest` to run the client side tests
* coverage should have increased after it's run
#### Any background context you want to provide?
n/a
#### What are the relevant pivotal tracker stories?(if applicable)
#157930865
